docs/operators.md

```diff
@@ -26,7 +26,11 @@
 * Indent/Outdent/Auto-indent
   * `vw>` - works in visual mode
   * `>>` - indent current line one level
+  * `>2>` - repeated linewise
+  * `2>>` - repeat indent operation
   * `<<` - outdent current line one level
+  * `<2<` - repeated linewise
+  * `2<<` - repeat outdent operation
```

Comment:
I tried it and in normal mode `>2>` behaves the same as `2>>` - indenting two lines once; that's consistent with VIM. The docs above seem to imply this PR was introducing a difference there - is it?